pkg/bpf/map.go

```diff
@@ -216,6 +217,57 @@ func (m *Map) setPathIfUnset() error {
 	return nil
 }
 
+func (m *Map) migrate(fd int) (bool, error) {
```

Comment:
In the future, migrate can actually perform migration. We are not maintaining backwards compatibility yet though. Still confusing?